tests/tests_pytorch/deprecated_api/test_remove_2-0.py

```diff
@@ -88,3 +88,8 @@ def test_v2_0_resume_from_checkpoint_trainer_constructor(tmpdir):
         trainer = Trainer(resume_from_checkpoint="trainer_arg_path")
     with pytest.raises(FileNotFoundError, match="Checkpoint at fit_arg_ckpt_path not found. Aborting training."):
         trainer.fit(model, ckpt_path="fit_arg_ckpt_path")
+
+
+def test_deprecated_amp_level():
```

Comment:
There is no date for 2.0 yet. So it's safer to set the removal with `1.10`

Reply:
done!